### ISSUE TITLE
Request to share the TJ4DRadSet data set link and test set label again

### ISSUE THREAD

**Issue by solitude-cheval** [TJRadarLab/TJ4DRadSet#18]
Dear TJ4DRadSet dataset holders,

I sincerely hope you are all well, maybe you are busy and didn't see my emails. Anyway first of all, I would like to appreciate you for sharing the link of TJ4DRadSet dataset to me earlier, it is very helpful for my research work. However, due to haste and carelessness, I only saved 3 of the network disk links, and the fourth link had expired when I tried to access it. I am very sorry to cause you trouble.

I also noticed that in the TJ4DRadSet Public1:4D Radar dataset, only the labels of the training set are included, but not the labels of the test set. I didn't keep a backup of my tracking tags before, and now that the link is dead, I can't get that information. In my research, the labels of the test set are crucial for model validation and performance evaluation.

Therefore, I sincerely request if you can share the full link of the TJ4DRadSet dataset again, and hope that this time I can save all the links properly to ensure the completeness and accuracy of the data. Also, if possible, I'd really like to be able to get the label information for the test set portion of this dataset so I can do a full experiment and analysis.

I know your time is valuable, but I also feel that this data is vital to my research. I would be very grateful if you could help me out with this again.

Finally, thank you again for your generous help and support. Looking forward to your reply.

**Comment by betterwyl** [comment]
I have also encountered this problem. The dataset I received only includes labels for the training data and not for the testing data.

**Comment by Manishnayak234** [comment]
Hi Solitude can you share the dataset you have available.

**Comment by TJRadarLab** [comment]
> Dear TJ4DRadSet dataset holders,
> 
> I sincerely hope you are all well, maybe you are busy and didn't see my emails. Anyway first of all, I would like to appreciate you for sharing the link of TJ4DRadSet dataset to me earlier, it is very helpful for my research work. However, due to haste and carelessness, I only saved 3 of the network disk links, and the fourth link had expired when I tried to access it. I am very sorry to cause you trouble.
> 
> I also noticed that in the TJ4DRadSet Public1:4D Radar dataset, only the labels of the training set are included, but not the labels of the test set. I didn't keep a backup of my tracking tags before, and now that the link is dead, I can't get that information. In my research, the labels of the test set are crucial for model validation and performance evaluation.
> 
> Therefore, I sincerely request if you can share the full link of the TJ4DRadSet dataset again, and hope that this time I can save all the links properly to ensure the completeness and accuracy of the data. Also, if possible, I'd really like to be able to get the label information for the test set portion of this dataset so I can do a full experiment and analysis.
> 
> I know your time is valuable, but I also feel that this data is vital to my research. I would be very grateful if you could help me out with this again.
> 
> Finally, thank you again for your generous help and support. Looking forward to your reply.

We released it a long time ago.
